client/app/index.html

```diff
@@ -15,6 +15,7 @@
     <!-- build:css css/styles.css -->
     <link rel="stylesheet" href="lib/css/fontawesome.css" />
     <link rel="stylesheet" href="lib/css/regular.css" />
+    <link rel="stylesheet" href="css/angular-datepicker.css" />
```

Comment:
Refusal from initial implementation.

I'm going to remove this myself while integrating the pull request.